### PR TITLE
Document updated log rolling directives + defaults

### PR DESCRIPTION
**Pull request by julianvmodesto**
For https://github.com/mholt/caddy/issues/1372 & https://github.com/mholt/caddy/pull/1379

### CODE REVIEW

File: content/docs/log.md [hunk 1]
```diff
@@ -46,19 +46,19 @@ Currently there are two predefined formats.
 
 ### Log Rotation
 
-If you enable log rotation, log files will be automatically maintained when they get large or old. You can use rotation by opening a block on your first line, which can be any of the variations described above:
+Log rotation is enabled by default. Log files will be automatically maintained when they get large or old.
```
Comment (mholt) [inline content/docs/log.md]: I realized this was worded wrong. Log files don't get rotated when they get old, but it's how long old (rotated) log files are kept before being deleted. The phrasing of the docs should be revised to say that instead.

File: content/docs/log.md [hunk 2]
```diff
@@ -46,19 +46,19 @@ Currently there are two predefined formats.
...
-	}
+    <span class="hl-subdirective">rotate_size</span> <i>maxsize</i>
+    <span class="hl-subdirective">rotate_age</span>  <i>maxage</i>
+    <span class="hl-subdirective">rotate_keep</span> <i>maxkeep</i>
 }</code>
 
 *   **maxsize** is the maximum size of a log file in megabytes (MB) before it gets rotated. Default is 100 MB.
-*   **maxage** is the maximum age of a rotated log file in days, after which it will be deleted. Default is to never delete old files because of age.
-*   **maxkeep** is the maximum number of rotated log files to keep. Default is to retain all old log files.
+*   **maxage** is the maximum age of a rotated log file in days, after which it will be deleted. Default is to delete old files after 14 days.
```
Comment (mholt) [inline content/docs/log.md]: This phrasing about the age parameter is correct.

File: content/docs/log.md [hunk 3]
```diff
@@ -74,12 +74,10 @@ Predefined format:
...
+With custom rotation:
 
 <code class="block"><span class="hl-directive">log</span> <span class="hl-arg">access.log</span> {
-	<span class="hl-subdirective">rotate</span> {
-		<span class="hl-subdirective">size</span> 100 <span class="hl-comment"># Rotate after 100 MB</span>
-		<span class="hl-subdirective">age</span>  14  <span class="hl-comment"># Keep log files for 14 days</span>
-		<span class="hl-subdirective">keep</span> 10  <span class="hl-comment"># Keep at most 10 log files</span>
-	}
+    <span class="hl-subdirective">rotate_size</span> 200 <span class="hl-comment"># Rotate after 200 MB; the default is 100 MB</span>
+    <span class="hl-subdirective">rotate_age</span>  30  <span class="hl-comment"># Keep log files for 30 days; the default is 14 days</span>
```
Comment (mholt) [inline content/docs/log.md]: Keep _old_ log files...

**Comment by julianvmodesto** [comment]
Made the requested changes to log.md!

Also tried adding clarification to errors.md.

**Comment by mholt** [comment]
Great! Thank you so much.

**Pull request merged**